pkg/operator/ceph/csi/spec.go

```diff
@@ -556,17 +556,28 @@ func createCSIDriverInfo(clientset kubernetes.Interface, name string, ownerRef *
 		},
 	}
 	csidrivers := clientset.StorageV1beta1().CSIDrivers()
-	k8sutil.SetOwnerRef(&csiDriver.ObjectMeta, ownerRef)
-	_, err := csidrivers.Create(csiDriver)
+	driver, err := csidrivers.Get(name, metav1.GetOptions{})
 	if err == nil {
-		logger.Infof("CSIDriver object created for driver %q", name)
-		return nil
-	}
-	if apierrors.IsAlreadyExists(err) {
+		if len(driver.ObjectMeta.OwnerReferences) > 0 {
+			// remove the owner reference set by operator on the csidriver
```

Comment:
How about adding the reference to #6162? It's to clarify why this block is necessary.